worlds/bk_sudoku/docs/en_Sudoku.md

```diff
@@ -0,0 +1,9 @@
+# Bk Sudoku
+
+## What is this game?
+
+Bk Sudoku is not like your avarage Archipelago game, it is instead a generic Sudoku client that can connect to an existing multiworld. When connected to an MultiWorld you can play the game to unlock random hints for your game. While slow and not very effective it will give you something to do when you have no checks left in your game.
```

Comment:
average*
a multiworld*

does clearing a puzzle give you an item hint or hint points? and if it's the former is it possible for it to hint for something you already have?

Reply:
it hints an missing location in world you are connected to, it can hint the same location multiple times if its still missing

Reply:
Maybe add that to the description here then as explicitly as possible